structure_tests/structure_test_v1.go

```diff
@@ -29,6 +29,7 @@ type StructureTestv1 struct {
 	CommandTests       []CommandTestv1
 	FileExistenceTests []FileExistenceTestv1
 	FileContentTests   []FileContentTestv1
+	LicenseTests       []string
```

Comment:
hmm, could you make this an actual struct?

What if we have a few different kinds of license tests:
debian (does what you have now)
file (you pass in a list of other LICENSE files to check, like if you wget or curl install something)